src/test/java/com/digitalasset/testing/SetupAppCallbackIT.java

```diff
@@ -0,0 +1,50 @@
+package com.digitalasset.testing;
+
+import com.daml.ledger.javaapi.data.*;
+import com.digitalasset.daml_lf.DamlLf1;
+import com.digitalasset.testing.junit4.Sandbox;
+import com.google.protobuf.InvalidProtocolBufferException;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExternalResource;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import static com.digitalasset.testing.Dsl.*;
+import static com.spotify.hamcrest.optional.OptionalMatchers.optionalWithValue;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class SetupAppCallbackIT {
+  private static final Path RESOURCE_DIR = Paths.get("src", "test", "resources").toAbsolutePath();
```

Comment:
Just a minor note, pls feel free to ignore: you may want to move the these config fields into a common class to avoid code dup.